research.html

```diff
@@ -1,55 +1,54 @@
 <!DOCTYPE html>
-<html lang="en" data-theme="light">
+<html lang="en">
+
 <head>
-  <title>John Doe | Research</title>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta name="keywords"
-    content="free portfolio template,portfolio template, portfolio website template, open source, open sourcing, contributing to open source" />
-  <meta name="description"
-    content="This project serves free and beautiful portfolio template for everyone. The sole aim of this project is to help all those who are not sure or face difficulties while trying to create the perfect portfolio for themselves. Anyone can showcase their work including moocs, education, projects, experience,etc. to the world." />
+    <title>John Doe | Research</title>
+    <meta charset="UTF-8">
```

Comment:
Avoid indentation and linting related changes for now. Else others would face merge conflicts